OKlib.js

```diff
@@ -824,416 +767,378 @@ registerPlugin({
+     * @param {Object[]} array the array to filter
+     * @param {String} attribute the attribute to filter for
+     * @param {Object} value value to check for
+     * @param {Boolean} isFunction flag for comparing function or attributes of objects
+     * @param {Function} compare a compare function that should be used for the comparison | if not set, 'equal' is used
+     * @return {Object[]} found object that matched the value
      */
     function arrayFilterByAttribute(array, attribute, value, isFunction, compare) {
-        if (!compare) {
-            compare = equal;
-        }
+        if (!compare) compare = equal;
         let result = [];
-        array.forEach(element => {
+        array.forEach(e => {
             if (isFunction) {
-                if (compare(element[attribute](), value)) {
-                    result.push(element);
-                }
+                if (compare(e[attribute](), value)) result.push(e);
             } else {
-                if (compare(element[attribute], value)) {
-                    result.push(element);
-                }
+                if (compare(e[attribute], value)) result.push(e);
             }
-        })
+        });
         return result;
     }
 
     /**
-     * Compares an objects property with a specific value
-     * @param  {Object} object   Object to check
-     * @param  {String} property Name of the Property that should be checked
-     * @param  {Object} element  Value that should be compared to the Property
-     * @param  {Function} compare    A Compare Function that should be used for the Comparison, if not set 'equal' is used
-     * @return {Boolean}   Returns the Value of the Comparison
+     * compares an objects property with a specific value
+     * @param {Object} object object to check
+     * @param {String} property name of the property that should be checked
+     * @param {Object} element value that should be compared to the property
+     * @param {Function} compare a compare function that should be used for the comparison | if not set, 'equal' is used
+     * @return {Boolean} returns the value of the comparison
      */
     function objectFunctionEqualsElement(object, property, element, compare) {
-        if (!compare) {
-            compare = equal;
-        }
+        if (!compare) compare = equal;
         return compare(object[property](), element);
     }
 
     /*
         Lib Definition
     */
 
-    const libModule = {
+    var libModule = {
         log: log,
 
         general: {
             checkVersion: checkVersion,
-            checkForUpdates: checkForUpdates,
             log: log,
-            getBots: getActiveBotInstances,
+            getBots: getActiveBotInstances
         },
 
         channel: {
             equal: equalChannelObjects,
             toString: channelToString,
             getDefault: channelGetDefault,
             getChannels: channelGetChannels,
-            getSubchannels: channelGetSubChannels, //legacy
```

Comment:
This might break scripts. So better to still include it? But on the other hand, only 1 or 2 scripts from Cedrik(Tuetchen)

Reply:
I added it back in just in case
I always prefer removing legacy shit.